release/ml_user_tests/horovod/horovod_user_test.py

```diff
@@ -18,7 +18,7 @@
     main(
         num_workers=6,
         use_gpu=True,
-        placement_group_timeout_s=900,
+        placement_group_timeout_s=2000,
```

Comment:
This looks like a recent degradation? While workaround is good, should we also surface the issue to Core?

Reply:
I think it was always flaky to begin with, and this is more of a product issue with nodes taking too long to start up right?

Reply:
I see. If that's the case, should we then add wait_for_cluster step?